common/metrics/defs.go

```diff
@@ -528,6 +531,9 @@ const (
 	RemoveEngineForShardLatency
 	CompleteDecisionWithStickyEnabledCounter
 	CompleteDecisionWithStickyDisabledCounter
+	HistoryEventNotificationQueuingLatency
```

Comment:
nit: misspelled queueing

Reply:
ok

however it seems that queuing is more common?
https://english.stackexchange.com/questions/60852/queueing-or-queuing